modules/core/src/test/kotlin/datamaintain/core/report/ReportTest.kt

```diff
@@ -22,6 +26,24 @@ internal class ReportTest {
         testAppender.start()
     }
 
+    /// Report fixture, containining everything
+    private fun reportFixture(): Report =
```

Comment:
I'm sorry I don't quite understand how this is a report fixture. It appears that a fixture is an equipment of some sort and I don't understand the link with what this method does. To me, it's just a function to build a report that will later be used in the tests, thus ```buildReport``` should be a good name. Since this function is only used by the nested tests in ```ExecuteLogs```,  it could be in that class as well.

Reply:
I was thinking along the lines of a test fixture - see https://github.com/junit-team/junit4/wiki/Test-fixtures for an explanation. But renaming it to buildReport() is fine with me - I don't really care either way.

Reply:
Oh ok, I did not know the name for that idea, thank you for giving me a link explaining it. Still, this method is not used to ensure that the tests always have the same result, no matter the repetition thus I prefer it named ```buildReport```